### PR TITLE
CDS-647: show builder tags on builder-page

### PR DESCRIPTION
**Pull request by warcholprzemo**
Tags on builder-page
tags-builderpage

### CODE REVIEW

**Comment by campos-ddc** [comment]
I've though about this again, and I think that it would be better to have tags only in the builder page (not the build page).

For example, I fear that it could be a bit confusing if you look at an individual build that has the tags `Unstable`, `Nightly`, `QV`. Does that mean that this particular build is Unstable because it failed? Was it triggered automatically as part of a Nightly thing? Was it run through QV?

The truth is that those tags are only saying something about the overall builder, not this particular build. It is Unstable because we don't guarantee that it will always pass, it is Nightly because it sometimes runs automatically, it is QV because it is a dependency of the QV build; but those conditions aren't always true for all builds.

@madski-unity what do you think about this?

**Comment by madski-unity** [comment]
On the page of a build, some parts of the page is telling what builder it
is a part of. I think the information is relevant there. One use case could
be "now I got this build failing on my branch - is it in QV? oh no ... it
is tagged unstable, and thus probably not caused by my changes". That could
be when someone runs zAllGraphicsTests and ends up looking at a single
failing build. Another relevant case is "I ran ABV but QV complained about
this failure - why is that? Oh - it is QV stable but not in ABV".

But agreed, it must be made clear that it is about "the builder", not "the
build". Ideally, "the build" could show "this failure has been seen as
instability this and that often across branches" based on hoarder or
something.

No big loss on the build page, as long as it is shown on the builder page.

(Slightly related: On https://ono.unity3d.com/unity/unity/www/trunk , I
"always" show the relevant tags next to the builder name. I think that
gives a relevant context.)

/Mads

**Comment by madski-unity** [comment]
But agreed, looking at the "sadness test 14": showing "builder details"
between information related to the particular build is not good. It would
be better without it.

**Comment by warcholprzemo** [comment]
@madski-unity  @campos-ddc 
I remove build tags from build-page.
I left tags only on builder-page.

**Review by campos-ddc — APPROVED** [review]

File: master/buildbot/status/web/build.py [hunk 1]
```diff
@@ -382,7 +382,6 @@ def content(self, req, cxt):
         cxt['chained_build'] = yield req.site.buildbot_service.master.db.buildrequests.getBuildChain(
             self.build_status.buildChainID,
         )
-
```
Comment (Mowinski) [inline master/buildbot/status/web/build.py]: I think we shouldn't include that changes in PR :)

File: master/buildbot/status/web/builder.py [hunk 2]
```diff
@@ -395,7 +396,6 @@ def prop_match(oprops):
         cxt['instant_json']["start_slaves"] = self.getSlavesJsonResource(filters, url, startslaves_dict)
 
         cxt['numbuilds'] = int(req.args.get('numbuilds', [self.numbuilds])[0])
-
```
Comment (Mowinski) [inline master/buildbot/status/web/builder.py]: The same :)

File: master/buildbot/status/web/builder.py [hunk 3]
```diff
@@ -634,7 +634,6 @@ def content(self, req, cxt):
                                                "stepFinished": filters,
                                            }}
 
-
```
Comment (Mowinski) [inline master/buildbot/status/web/builder.py]: The same

**Review by Mowinski — APPROVED** [review]

File: master/buildbot/status/web/build.py (hunk 1 above)
Comment (warcholprzemo) [reply master/buildbot/status/web/build.py]: Done

File: master/buildbot/status/web/builder.py (hunk 2 above)
Comment (warcholprzemo) [reply master/buildbot/status/web/builder.py]: Done

File: master/buildbot/status/web/builder.py (hunk 3 above)
Comment (warcholprzemo) [reply master/buildbot/status/web/builder.py]: Done

File: www/templates/builder.html [hunk 4]
```diff
@@ -31,7 +31,17 @@
     <div class="head-2">{{ description }}</div>
 
     {% endif %}
-     
+
+      <h2 class="head-2">
```
Comment (Mowinski) [inline www/templates/builder.html]: One finding, what about a situation, when there won't tags? Should we still display this header?

Comment (campos-ddc) [reply www/templates/builder.html]: Probably no

Comment (warcholprzemo) [reply www/templates/builder.html]: Done

**Pull request merged**